### DISCUSSION TITLE
We may have alerts from Kibana on the Alerts tab in Security Onion?

### DISCUSSION THREAD

**Discussion by ViByy** [Security-Onion-Solutions/securityonion#4349] [Unsupported Versions] [answered]
Hello,

Please help me with a piece of information regarding Alerts from Security Onion.
I need to know if it's possible to receive Kibana alerts (Ossec, Syslog) on the Alerts tab from Security Onion?
I have alerts from Kibana, but I want to know if it is possible to receive those alerts in the Alerts tab of Security Onion because I want to send that alerts from the Alerts tab to TheHive.

Thank you in advance.

Best regards,
ViByy

**Comment by weslambert** [comment, marked as answer]
If the event is of `event.dataset:alert`, then it will be presented in the Alerts tab.  Kibana and Alerts/Hunt are both looking at the same backend data, it's just that the `Alerts` view filters it to only those with the above referenced trait(s).

**Reply by ViByy** [reply]
Thanks a lot, Weslambert.
My issue was from NiFi, there was some misconfiguration regarding some query. 
In NiFi I had a different query alert than what was present in Security Onion.
